content/en/usage/common-problems.md

```diff
@@ -1,7 +1,258 @@
+
+However, it is also possible that the app does not support your phone's architecture at all and no compatible version exists. In this case, you will unfortunately have to use a different device or emulator to run the app.
+
+{{< hint info >}}
+**Note**: If you are running apps in an Android emulator, we recommend using Android 11 x86_64 emulators for the best compatibility. This particular combination supports apps compiled for all relevant architectures: x86_64, x86, arm64-v8a, armeabi-v7a, armeabi. Notably, it can also [run ARM apps with good performance](https://android-developers.googleblog.com/2020/03/run-arm-apps-on-android-emulator.html).  
+This unfortunately [does not appear to be the case for newer Android versions](https://github.com/tweaselORG/appstraction/issues/54).
+{{< /hint >}}
+
+## Only one app's traffic is recorded by `cyanoacrylate`
+
+When recording network traffic with `cyanoacrylate` or `tweasel-cli`, you may notice that the HAR file only contains traffic from the app you ran an analysis on but not from other apps or the system. This is expected. By default, both `cyanoacrylate` and `tweasel-cli` only record the traffic of the specified app to facilitate attribution. But you can change this behaviour.
+
+In `cyanoacrylate`, if you run `startTrafficCollection()` on an `AppAnalysis`, you will only ever get this app's traffic. However, you can also start a traffic collection on an `Analysis` and precisely specify which apps' traffic you want to record:
+
+```ts
+// Collect traffic from all apps, including the system.
+analysis.startTrafficCollection({ mode: 'all-apps' });
+// Collect traffic only from the specified apps.
+analysis.startTrafficCollection({
+    mode: 'allowlist',
+    apps: ['com.example.app1', 'com.example.app2'],
+});
+// Collect traffic from all apps except the specified apps.
+analysis.startTrafficCollection({
+    mode: 'denylist',
+    apps: ['com.example.app1', 'com.example.app2'],
+});
+```
+
+In `tweasel-cli`, you can use the `--all-traffic` option to record all traffic:
+
+```sh
+tweasel record-traffic app.apk --all-traffic
+```
+
+{{< hint warning >}}
+**Note**: App filtering is currently only supported on Android. On iOS, we always record all traffic.
+{{< /hint >}}
+
+## Recording is missing traffic due to problems with certificate pinning bypass
+
+Some apps use a technique called certificate pinning to verify the identity of the servers they communicate with. This means that they only accept connections from servers that present a specific certificate that the app knows in advance. This is a security measure to prevent machine-in-the-middle attacks, but it also prevents us from recording the app's traffic by using a proxy with our own certificate.
+
```

Comment:
Maybe mention here how to recognize cert pinning issues using the mitmproxy events.